### ISSUE TITLE
Moccasin GUI fails to run on MacOS Sierra

### ISSUE THREAD

**Issue by mhucka** [sbmlteam/moccasin#15]
A user reported the following error, and it is repeatable in my environment too:

```
12 clarity> ./moccasin_GUI.py                                                                  moccasin/moccasin/interfaces
./moccasin_GUI.py:182: wxPyDeprecationWarning: Call to deprecated item. Use SetSizeHints instead.
  self.SetSizeHintsSz( wx.Size( 760,-1 ))
Traceback (most recent call last):
  File "./moccasin_GUI.py", line 572, in <module>
    gui_main()
  File "./moccasin_GUI.py", line 568, in gui_main
    frame = MainFrame(None)
  File "./moccasin_GUI.py", line 189, in __init__
    self.statusBar = self.CreateStatusBar(5, wx.ST_SIZEGRIP|wx.ALWAYS_SHOW_SB|wx.RAISED_BORDER, wx.ID_ANY )
AttributeError: 'module' object has no attribute 'ST_SIZEGRIP'
```

**Comment by mhucka** [comment]
Fixed by PR #16.

**Comment by mhucka** [comment]
Fixed by PR #16.
